src/rules/onlyArrowFunctionsRule.ts

```diff
@@ -57,24 +57,44 @@ export class Rule extends Lint.Rules.AbstractRule {
 }
 
 class OnlyArrowFunctionsWalker extends Lint.RuleWalker {
+    private allowDeclarations: boolean;
+    private allowNamedFunctions: boolean;
+
+    constructor(sourceFile: ts.SourceFile, options: Lint.IOptions) {
+        super(sourceFile, options);
+        this.allowDeclarations = options.ruleArguments.indexOf(OPTION_ALLOW_DECLARATIONS) !== -1;
```

Comment:
`options` is already in scope, so why not use it.

I don't have a strong opinion on this one